website/src/pages/ComparePage/ComparePage.tsx

```diff
@@ -80,65 +72,17 @@ export default function Compare() {
                   <Macrobench
                     data={macro}
                     gitRef={{
-                      left: gitRef.left.slice(0, 8),
-                      right: gitRef.right.slice(0, 8),
+                      old: gitRef.old.slice(0, 8),
+                      new: gitRef.new.slice(0, 8),
                     }}
-                    commits={{ left: gitRef.left, right: gitRef.right }}
+                    commits={{ old: gitRef.old, new: gitRef.new }}
                   />
                 </div>
               );
             })}
           </div>
         </section>
       )}
-
-      {/*{!isMicrobenchLoading && dataMicrobench && (*/}
```

Comment:
Could we delete this?

Reply:
Yes, this whole comment was not used and not maintained. We disabled **micro** benchmarks a while ago. We should re-think them entirely if we want to re-implement them. This is just some clean up deletion.